coders/webp.c

```diff
@@ -249,14 +259,32 @@ static int ReadSingleWEBPImage(Image *image,const uint8_t *stream,
   MagickBooleanType
     status;
 
+  if (is_first) {
+    canvas_width = image->columns;
+    canvas_height = image->rows;
+    x_offset = image->page.x;
+    y_offset = image->page.y;
+    image->page.x = 0;
+    image->page.y = 0;
+  } else {
+    x_offset = 0;
+    y_offset = 0;
+  }
   webp_status = FillBasicWEBPInfo(image,stream,length,configure);
+  image_width = image->columns;
+  image_height = image->rows;
+  if (is_first) {
```

Comment:
We have a different coding convention for `if` statements. Could you change it to this:

```c
if (is_first)
  {
    image->columns = canvas_width;
    image->rows = canvas_height;
  }
```

There are more statements in this PR so please change all of them.